superset-frontend/plugins/legacy-preset-chart-nvd3/src/NVD3Controls.tsx

```diff
@@ -170,7 +170,7 @@ export const showControls: CustomControlItem = {
     default: false,
     description: t(
       'Whether to show extra controls or not. Extra controls ' +
-        'include things like making mulitBar charts stacked ' +
+        'include things like making multiBar charts stacked ' +
```

Comment:
this might actually need to be multi-bar... but what you have here is an improvement, so that's fine :D